tests/foreman/api/test_contentviewversion.py

```diff
@@ -163,8 +163,15 @@ def test_incremental_update_puppet(self):
         ).create()
         with open(get_data_file(PUPPET_MODULE_NTP_PUPPETLABS), 'rb') as handle:
             puppet_repo.upload_content(files={'content': handle})
+        # Extract all the available puppet modules.
+        puppet_modules = content_view.available_puppet_modules()['results']
+        # Make sure that we have results. Uploading content does not
+        # seem to create a task so we cannot pool it for status. We
+        # should then check that we have some results back before
+        # proceeding.
+        self.assertGreater(puppet_modules, 0)
```

Comment:
Doh again! I had `len` there but then did some other changes and removed it :/  No karma for me